components/fixme-page.jsx

```diff
@@ -0,0 +1,13 @@
+var React = require('react');
+
+var PlaceholderPage = require('./placeholder-page.jsx');
+
+var FixmePage = React.createClass({
```

Comment:
This is a great idea! In the future maybe we can add some kind of "pre-production-test" to ensure that we have no links pointing here before we push a new release. :)